package.json

```diff
@@ -10,17 +10,20 @@
   },
   "dependencies": {
     "@daohaus/abi-utilities": "^0.0.2",
-    "@daohaus/common-utilities": "^0.0.12",
+    "@daohaus/common-utilities": "file: ./../../../daohaus-monorepo/dist/libs/common-utilities",
```

Comment:
only change here was adding `"esModuleInterop": true` then linking the common-utils built locally from that